src/Shared/E2ETesting/SeleniumStandaloneServer.cs

```diff
@@ -178,7 +178,13 @@ private static async Task InitializeInstance(ITestOutputHelper output)
         // Log
         void LogOutput(object sender, DataReceivedEventArgs e)
         {
-            logOutput.TryAdd(e.Data);
+            try
+            {
+                logOutput.TryAdd(e.Data);
```

Comment:
This was failing because the collection was already completed and could not log anything. It was causing the test run to crash.